src/main/java/org/terasology/alterationEffects/regenerate/RegenerationAlterationEffect.java

```diff
@@ -53,6 +54,7 @@ public RegenerationAlterationEffect(Context context) {
      * @param magnitude     The magnitude of the regen effect.
      * @param duration      The duration of the regen effect.
      */
+    @SuppressWarnings("CheckStyle")
```

Comment:
Ideally this shouldn't be included. We usually want to fix rather than suppress checkstyle issues

Reply:
This was unintentional, I thought suppressing some issues will not show up in code. I will remove this in next commit.